examples/python/qiskit_integration.py

```diff
@@ -68,23 +68,14 @@ def vqe_solve(op, maxiter, quantum_instance):
 #
 # We can now run the VQE algorithm. In this example we use only 50 iterations, but greater accuracy may be achieved by increasing this number:
 
+vqe_solve(H2_op, 50, qi)
+
 print("VQE result:", vqe_solve(H2_op, 50, qi))
```

Comment:
Seems we are calling `vqe_solve()` twice,

Reply:
Fixed, thanks.